### PR TITLE
Bugfix - Check window for node environments

### PR DESCRIPTION
**Pull request by rmi22186**

### CODE REVIEW

File: src/GoogleAdWordsEventForwarder.js [hunk 1]
```diff
@@ -74,7 +74,7 @@
             if (window.google_trackConversion) {
                 window.google_trackConversion(adWordEvent);
             } else {
-                eventQueue.push(event);
+                eventQueue.push(adWordEvent);
```
Comment (alexs-mparticle) [inline src/GoogleAdWordsEventForwarder.js]: Was this intended to be part of a different PR?

Comment (rmi22186) [reply src/GoogleAdWordsEventForwarder.js]: it was just something i caught when removing window.  i can make it a diff PR tho

**Review by alexs-mparticle — APPROVED** [review]
